src/pcapi/core/offers/repository.py

```diff
@@ -206,6 +206,20 @@ def get_offers_map_by_id_at_providers(id_at_providers: list[str]) -> dict[str, i
     return offers_map
 
 
+def get_offers_map_by_venue_reference(id_at_providers: list[str], venue_id: int) -> dict[str, int]:
+    from pcapi.core.offers.api import compute_venue_reference
+
+    offers_map = {}
+    for offer_id, offer_id_at_provider in (
+        db.session.query(Offer.id, Offer.idAtProvider)
+        .filter(and_(Offer.venueId == venue_id, Offer.idAtProvider.in_(id_at_providers)))
+        .all()
+    ):
+        offers_map[compute_venue_reference(offer_id_at_provider, venue_id)] = offer_id
+
+    return offers_map
```

Comment:
Pourquoi on a besoin d'un dict ici ?
Je ne vois pas offer_id utilisé après

```suggestion
    offers_map = {}
    for offer_id_at_provider in (
        db.session.query(Offer.idAtProvider)
        .filter(and_(Offer.venueId == venue_id, Offer.idAtProvider.in_(id_at_providers)))
        .all()
    ):
        offers_map.add(compute_venue_reference(offer_id_at_provider, venue_id))
```

Reply:
oui c'est vrai